reactive/docker.py

```diff
@@ -85,14 +86,14 @@ def proxy_changed():
 
 
 def install_from_archive_apt():
-    status_set('maintenance', 'Installing docker.io from apt archive')
+    status_set('maintenance', 'Installing docker.io from universe.')
```

Comment:
I like the clarity on this messaging in how we refactored the message to the end user what its doing